examples/private_network_existing_network/variables.tf

```diff
@@ -80,4 +80,4 @@ variable "tiering_ssd_percent" {
 variable "cluster_size" {
   type = number
   description = "Weka cluster size"
-}
```

Comment:
no reason for change here and the other places with "\n"

Reply:
fixed